docs/guide/topics/error_handling.rst

```diff
@@ -0,0 +1,361 @@
+
+        if isinstance(error, commands.MissingPermissions):
+            formatted_perms = ', '.join(error.missing_perms)
+            await ctx.send(f'You are missing the following permissions: {perms}')
+
+        elif isinstance(error, commands.BotMissingPermissions):
+            formatted_perms = ', '.join(error.missing_perms)
+            await ctx.send(f'I am missing the following permissions: {perms}')
+
+
+.. warning::
+    After the local error handlers get called, the global error handler gets called too. If you're handling unhandled errors 
+    in it, do not handle them in local error handlers, as this may cause them to be handled twice or more.
+
+Using Global and Local Error Handlers Together
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Some times, you may want or need to use local and global error handlers, in these cases, there are multiple ways of checking if 
+the error has already been handled before. There are many approaches on how to do this:
+
+- Using the :meth:`Cog.has_error_handler <discord.ext.commands.Cog.has_error_handler>`
+  and :meth:`Command.has_error_handler <discord.ext.commands.Command.has_error_handler>`
+  methods in your global error handler.
+
+**Example**
+
+.. code-block:: python3
+
+    @bot.listen()
+    async def on_command_error(ctx, error):
+        """ Global error handler that ignores local error handlers. """
+
+        if ctx.cog and ctx.cog.has_error_handler():
+            # If the cog has an error handler, we return,
+            # ignoring the error.
+            return
+        
+        if ctx.command and ctx.command.has_error_handler():
+            # We do the same for the command.
+            return
+        
+        # Simpler way to handle CommandInvokeError using getattr
+        error = getattr(error, 'original', error)
+
+        # Then we continue with our error handling as normal...
+        print(f"Ignoring unhandled exception in command {ctx.command!r}")
+        traceback.print_exception(type(error), error, error.__traceback__)
+
+- Attaching flags to the :class:`Context <~.ext.commands.Context>` to denote the error has been handled.
+
+**Example**
+
+*1. In the local error handler, we attach an attribute to our Context:*
+
+.. code-block:: python3
+
+    @ban.error
+    async def ban_error(ctx, error):
+        """ Handles only the MissingRequiredArgument error. """
+
+        if isinstance(error, commands.MissingRequiredArgument):
+            # We set an attribute to the context that will then
+            # be detected in the global error handler.
+            ctx.ignore_error = True
+
+            # Then we do the rest
+            await ctx.send(f'Please mention the person to ban using `{ctx.clean_prefix}ban @member`')
+
+*2. Then, in our error handler, we use ``getattr()`` to check for the custom attribute we just set:*
+
+..
+    Note, I couldn't get to reference getattr() to the python docs... >_<
```

Comment:
Someone else probably knows the answer to this but I'll elevate it for attention.

Reply:
```rst
*2. Then, in our error handler, we use* :func:`getattr` *to check for the custom attribute we just set:*
```
I think this solves it.
Found out that Sphinx can't parse ``` :func:`getattr` ``` and other references in a formatted string.